### ISSUE TITLE
Vanilla install of LXC + luci-lxc-app not working

### ISSUE THREAD

**Issue by tcuthbert** [openwrt/packages#2117]
Hi,

I have pulled the luci-lxc-app packages from the opkg repos but unable to get an openwrt container running successfully.

When I attempt to attach to a busybox container I get an error that an error that the init pid is incorrect.

Is there any documentation on how to set LXC up on openwrt?

**Comment by vancluever** [comment]
Just thought I'd mention something: There seems to have been no activity on this package in a year, and also the domain referenced here has now been squatted and redirects to what looks like sketchy stuff.

```
feeds/packages/utils/luci-app-lxc/files/lxc.config:6:	option url 'virtualwrt.org/containers/'
```

Is it safe to assume this package is dead?

**Comment by vladwing** [comment]
I just tried this package. The virutalwrt.org url is completely dead.

**Comment by localnet** [comment]
I did a pull request to luci-app-lxc for it use images.linuxcontainers.org.

If somebody want to test it: https://github.com/openwrt/luci/pull/1422

**Comment by dibdot** [comment]
Fixed in 18.06
